controllers/admin.js

```diff
@@ -1,29 +1,93 @@
 const adminService = require('~/services/admin')
 
-const addAdmin = async (req, res) => {
+const inviteAdmins = async (req, res) => {
+  const admins = await adminService.inviteAdmins({
+    ...req.body,
+    language: req.lang
+  })
 
+  res.status(201).json(admins)
 }
 
-const getAdmins = async (_req, res) => {
+const getInvitations = async (req, res) => {
+  const invitations = await adminService.getInvitations()
 
+  res.status(200).json(invitations)
+}
+
+const getAdmins = async (req, res) => {
+  const {
+    skip,
+    limit,
+    name,
+    email,
+    active,
+    blocked,
+    signUpDateFrom,
+    signUpDateTo,
+    lastLoginFrom,
+    lastLoginTo,
+    sortByName,
+    sortByEmail,
+    sortByLastLogin,
+    sortBySignUpDate
+  } = req.query
+
+  const admins = await adminService.getAdmins({
+    skip: parseInt(skip),
+    limit: parseInt(limit),
+    name,
+    email,
+    active: Boolean(parseInt(active)),
+    blocked: Boolean(parseInt(blocked)),
+    signUpDateFrom: new Date(signUpDateFrom),
+    signUpDateTo: new Date(signUpDateTo),
+    lastLoginFrom: new Date(lastLoginFrom),
+    lastLoginTo: new Date(lastLoginTo),
+    sortByName: parseInt(sortByName),
+    sortByEmail: parseInt(sortByEmail),
+    sortByLastLogin: parseInt(sortByLastLogin),
+    sortBySignUpDate: parseInt(sortBySignUpDate)
+  })
+
+  res.status(200).json(admins)
 }
 
 const getAdminById = async (req, res) => {
+  const admin = await adminService.getAdminById(req.params.id)
```

Comment:
we stick to destructuring assignment, don't we?

Reply:
ok